### ISSUE TITLE
Sounds of mods stop functionning after switching between to session on windows

### ISSUE THREAD

**Issue by Peperehobbits01** [noblereign/GX-Mods-Extension#17]
**Describe the issue**
Sounds of mods stop functionning after switching between to session on windows and not disconnecting while Firefox is open.

**To Reproduce**
1. Open Firefox on a Windows machine
2. Go in the starter menu of windows and switch of sessions.
3. Then go back to your sessions where your Firefox is opened.
4. All sounds of the extention/mods will not play.

**Expected behavior**
Well, the sounds to play after I come back from the other session.

**Screenshots**
Well it's sounds so... xD

**System Information**
 - OS: [e.g. Windows 10]
 - Browser [e.g. Firefox]
 - Version [e.g. 127.0.2]

**Additional context**
I think, I will create a pull request whit updated issues templates, to have something whit a bit more style, I'll work on it and create a pull request soon. Anyway, this bug isn't realy urgent to fix I guess but it's nice if it gets.

**Comment by noblereign** [comment]
commenting to let you know that i have seen this issue, currently working on implementing web modding and themes atm but once i'm done with that i'll see what i can do about this

**Comment by noblereign** [comment]
i did notice that the extension suspension code was horrendously flawed, i have no clue if said code even gets called by Firefox but the next release will fix up that code, maybe it'll help, maybe it won't. who knows?

btw, when you say switching sessions, do you mean switching user accounts or something different?

**Comment by Peperehobbits01** [comment]
> i did notice that the extension suspension code was horrendously flawed, i have no clue if said code even gets called by Firefox but the next release will fix up that code, maybe it'll help, maybe it won't. who knows?
> 
> btw, when you say switching sessions, do you mean switching user accounts or something different?

Yep switching user accounts, and thanks for the fast responds :wink:

**Comment by noblereign** [comment]
ohh you know what, i got a new laptop and i think i'm seeing something like this happening when i let it go to sleep for a long time. perhaps i *will* be able to debug this in the near future

**Comment by Peperehobbits01** [comment]
Well, it was on a laptop for me to.
